exchange/impcustomcachekeytest/multiImpCustomCacheKey.json

```diff
@@ -0,0 +1,100 @@
+{
+	"bidRequest": {
+		"imp": [{
+				"id": "oneImp",
+				"exp": 300
+			},
+			{
+				"id": "twoImp"
+			}
+		]
+	},
+	"pbsBids": [{
+		"bid": {
+			"id": "bidOne",
+			"impid": "oneImp",
+			"price": 7.64,
+			"exp": 600,
+			"cat": ["11_sports_22"]
+		},
+		"bidType": "banner",
+		"bidder": "appnexus"
+	}, {
+		"bid": {
+			"id": "bidTwo",
+			"impid": "oneImp",
+			"price": 15.64,
+			"exp": 200,
+			"cat": ["33_news_44"]
+		},
+		"bidType": "banner",
+		"bidder": "pubmatic"
+	}, {
+		"bid": {
+			"id": "bidThree",
+			"impid": "oneImp",
+			"price": 2.3,
+			"cat": ["33_news_44"]
+		},
+		"bidType": "banner",
+		"bidder": "openx"
+	}, {
+		"bid": {
+			"id": "bidFour",
+			"impid": "twoImp",
+			"price": 1.64,
+			"cat": ["11_sports_22"]
+		},
+		"bidType": "banner",
+		"bidder": "appnexus"
+	}, {
+		"bid": {
+			"id": "bidFive",
+			"impid": "twoImp",
+			"price": 7.64,
+			"exp": 900,
+			"cat": ["11_sports_22"]
+		},
+		"bidType": "banner",
+		"bidder": "rubicon"
+	}],
+	"expectedCacheables": [{
+			"Type": "json",
+			"TTLSeconds": 360,
+			"Key": "11_sports_22_abcd-efgh-ijkl-mnop",
+		    "Data": "{\"id\": \"bidOne\", \"impid\": \"oneImp\", \"price\": 7.64,  \"cat\": [\"11_sports_22\"],\"exp\": 600}"
+		}, {
+			"Type": "json",
+			"TTLSeconds": 260,
+			"Key": "33_news_44abcd-efgh-ijkl-mnop",
+		    "Data": "{ \"id\": \"bidTwo\", \"impid\": \"oneImp\", \"price\": 15.64, \"cat\": [\"33_news_44\"],\"exp\": 200 }"
+		}, {
+			"Type": "json",
+			"TTLSeconds": 360,
+			"Key": "33_news_44abcd-efgh-ijkl-mnop",
+		    "Data": "{ \"id\": \"bidThree\", \"impid\": \"oneImp\", \"price\": 2.3,\"cat\": [\"33_news_44\"] }"
+		},
+		{
+			"Type": "json",
+			"TTLSeconds": 360,
+			"Key": "11_sports_22_abcd-efgh-ijkl-mnop",
+		    "Data": "{ \"id\": \"bidFour\", \"impid\": \"twoImp\", \"price\": 1.64,\"cat\": [\"11_sports_22\"] }"
+		},
+		{
+			"Type": "json",
+			"TTLSeconds": 960,
+			"Key": "11_sports_22_abcd-efgh-ijkl-mnop",
+		    "Data": "{ \"id\": \"bidFive\", \"impid\": \"twoImp\", \"price\": 7.64, \"cat\": [\"11_sports_22\"],\"exp\": 900 }"
+		}
+	],
+	"defaultTTLs": {
+		"banner": 300,
+		"video": 3600,
+		"audio": 1800,
+		"native": 300
+	},
+	"targetDataIncludeWinners": true,
+	"targetDataIncludeBidderKeys": true,
+	"targetDataIncludeCacheBids": true,
+	"targetDataIncludeCacheVast": false
```

Comment:
As above for customcachkeytest files, these two values are wrong.

Reply:
You mean `"targetDataIncludeCacheBids": true` and `"targetDataIncludeCacheVast": false` ? Why are they wrong?

Reply:
Because for non-vast requests, we don't generate custom cache keys.

Reply:
We could but the outcome would not change at all. Do you want me to update the PR again?

Reply:
This file is still wrong as is.  In its current state, it's a repeat of cachetest/multibid.json but with expected key values that are wrong but never evaluated because of the current state of auction_test.go.  Reversing the last two lines would cause it to make sense and not be a repeat of cachetest/multibid.json, though it would probably now be necessary to change the Type values from json to xml (and maybe the bidTypes of the bids), and we would still be still missing the other associated test cases we've come to expect (i.e., tests toggling the values of the two lines above the last two lines).

Reply:
Also, if the file is to be kept as a test for custom cache keys for multiple imp requests, the category values of the bids should be unique, and that will affect the expected key values.  In addition, in that case, only the winning bids should have custom (expected) key values.

Reply:
The multibid test has been corrected. It still is a multi-imp, multi-bid all banner request. To test a multi-imp, multi-bid vast request, a new json file has been added so now we have two different tests in that folder: 
```
▾ exchange/
  ▸ cachetest/
  ▸ customcachekeytest/
  ▸ exchangetest/
  ▾ impcustomcachekeytest/
      multiImpBanners.json
      multiImpVast.json
```